schema/auth.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$id": "https://serverlessworkflow.io/schemas/0.7/auth.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Serverless Workflow specification - auth schema",
+  "type": "object",
+  "auth": {
+    "oneOf": [
+      {
+        "type": "string",
+        "format": "uri",
+        "description": "URI to a resource containing auth definitions (json or yaml)"
+      },
+      {
+        "type": "array",
+        "description": "Workflow auth definitions",
+        "items": {
+          "type": "object",
+          "$ref": "#/definitions/authdef"
+        },
+        "additionalItems": false,
+        "minItems": 1
+      }
+    ]
+  },
+  "required": [
+    "auth"
+  ],
+  "definitions": {
+    "authdef": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Unique auth name",
+          "minLength": 1
+        },
+        "type": {
```

Comment:
We should probably rename that to `scheme`, which is, IMHO, more ubiquitous

Reply:
done